packages/astro-integration/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@codecov/astro-integration",
+  "version": "1.4.0",
+  "description": "Official Codecov Astro integration",
+  "author": "Codecov",
+  "license": "MIT",
+  "homepage": "https://github.com/codecov/codecov-javascript-bundler-plugins/tree/main/packages/astro-integration",
+  "repository": "git://github.com/codecov/codecov-javascript-bundler-plugins.git",
+  "keywords": [
+    "withastro",
+    "astro-component",
+    "astro-integration",
+    "Codecov",
+    "Astro",
+    "bundler",
+    "analytics",
+    "optimization"
+  ],
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.cjs"
+      }
+    }
+  },
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "type-check": "tsc --noEmit",
+    "build": "unbuild",
+    "dev": "unbuild --stub && node --watch-path=src dist/index.mjs",
+    "clean": "rm -rf dist  node_modules",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "pnpm lint --fix",
+    "format": "prettier '**/*.{cjs,mjs,ts,tsx,md,json}' --ignore-path ../.gitignore --ignore-unknown --no-error-on-unmatched-pattern --write",
+    "format:check": "prettier '**/*.{cjs,mjs,ts,tsx,md,json}' --ignore-path ../.gitignore --ignore-unknown --no-error-on-unmatched-pattern --check",
+    "test:unit": "vitest run",
+    "test:unit:watch": "vitest watch",
+    "test:unit:ci": "vitest --coverage --reporter=junit --outputFile=./bundler-plugin-core.junit.xml run",
+    "test:unit:update": "vitest -u run",
+    "generate:typedoc": "typedoc --options ./typedoc.json"
+  },
+  "dependencies": {
+    "@codecov/bundler-plugin-core": "workspace:^",
+    "@codecov/vite-plugin": "workspace:^",
+    "unplugin": "^1.10.1"
+  },
+  "devDependencies": {
+    "@rollup/plugin-replace": "^5.0.5",
+    "@types/node": "^20.11.15",
+    "@vitest/coverage-v8": "^1.5.0",
+    "astro": "^4.16.14",
+    "codecovProdRollupPlugin": "npm:@codecov/rollup-plugin@0.0.1-beta.5",
```

Comment:
We can upgrade this version to `"npm:@codecov/rollup-plugin@1.5.0"` now to match the others